api-gateway/src/main/resources/static/scripts/auth/signup/signup-form.template.html

```diff
@@ -2,68 +2,177 @@
 <!--        * User: @JordanAlbayrak-->
 <!--        * Date: 22/09/21-->
 <!--        * Ticket: feat(AUTH-CPC-102)_signup_user-->
-<div style="display: flex; justify-content: center">
-    <div>
+<div style="display: flex; justify-content: center;">
+    <div style="width: 400px;">
         <h2 class="text-center">Signup</h2>
-        <h3 class="text-danger text-center" ng-repeat="error in errorMessages">{{ error }}</h3>
-        <div class="d-flex justify-content-center">
-            <form class="form-horizontal" ng-submit="$ctrl.add()" style="max-width: 25em; justify-content: center;">
```

Comment:
Why did you remove all of this ?

Reply:
I didn't I just modified it a bit and moved it around except the <h3>